### PR TITLE
wxGUI/lmgr: allow change group layer opacity level

### LINKED ISSUE #3376: Allow group selection opacity changes in g.gui [Feat]
**Is your feature request related to a problem? Please describe.**

It would really be nice to be able to control the opacity of a group selection of layers in g.gui all at once, much the same as the ability to check, uncheck, and remove a group selection. 

**Describe the solution you'd like**

Select a series of layers in the Layer panel of g.gui, right-click and select 'Change opacity level', which would bring up the same dialog box that you get when a single layer's opacity is changed. Except in this case, all of the selected layers would adopt the same opacity level that is chosen on the slider in the dialog box.

**Describe alternatives you've considered**

The only alternative is changing the opacity of each layer individually, which is extremely tedious is you have anything more than a few layers loaded in g.gui.

### PR DESCRIPTION
**Pull request by tmszi**
Implement and closes #3376.

### CODE REVIEW

File: gui/wxpython/lmgr/layertree.py [hunk 1]
```diff
@@ -578,6 +578,33 @@ def OnLayerContextMenu(self, event):
...
+            child_is_maplayer = None
+            if child:
+                while child:
+                    child_maplayer = self.GetLayerInfo(child, key="maplayer")
+                    child_ltype = self.GetLayerInfo(child, key="type")
+                    if child_maplayer and child_ltype != "command":
+                        child_is_maplayer = True
+                        break
+                    child = self.GetNextSibling(child)
+            if ltype == "group" and child_is_maplayer:
```
Comment (petrasovaa) [inline gui/wxpython/lmgr/layertree.py:594]: Wouldn't `ltype` be always group here?

File: gui/wxpython/lmgr/layertree.py [hunk 2]
```diff
@@ -1189,6 +1216,49 @@ def OnPopupProperties(self, event):
...
+            if maplayer and ltype != "command":
+                break
+            child = self.GetNextSibling(child)
+            if child is None:
+                child, cookie = self.GetNextChild(child, cookie)
+        current_opacity = maplayer.GetOpacity()
+        dlg = SetOpacityDialog(
+            self,
+            opacity=current_opacity,
+            title=_("Set opacity of <{}>").format(maplayer.GetName()),
```
Comment (petrasovaa) [inline gui/wxpython/lmgr/layertree.py:1235]: Could this have the name of the group instead? Right now it shows the name of the first layer.

File: gui/wxpython/lmgr/layertree.py (hunk 1 above)
Comment (tmszi) [reply gui/wxpython/lmgr/layertree.py:594]: Yes you are right.

File: gui/wxpython/lmgr/layertree.py (hunk 2 above)
Comment (tmszi) [reply gui/wxpython/lmgr/layertree.py]: Fixed with https://github.com/OSGeo/grass/pull/3583/commits/61c390e19ee2dbaeb9b678eacd1ae454b72a00d3.

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**